<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetTestItem</class>
 <widget class="QDialog" name="DialogSetTestItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>328</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Test Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sequence name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPresetName">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCommand">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Command</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelCommandName">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Command name</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditP2">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cmd to Reader 1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditINS">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Lc (Le) (auto)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditCommandName">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>P1</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditP1">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>P2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditCLA">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBoxCmdToDevice1">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CLA</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEditLEN">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>INS</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DATA</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEditDataField">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxAnswer">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Answer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelAnsLen">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Response size (without SW)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditAnsLen">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Response size min (without SW)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditAnsLenMin">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Response size max (without SW)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditAnsLenMax">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SW1 (HEX)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditSW1">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SW2 (HEX)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditSW2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelSave2Buf1">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save reponse to buf 1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkSave2Buf1">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelSave2Buf2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save reponse to buf 2</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkSave2Buf2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelLoadFromBuf1">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load data from buf 1</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="checkLoadFromBuf1">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelLoadFromBuf2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load data from buf 2</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="checkLoadFromBuf2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>USB reset</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="checkBoxResetUSB">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditPresetName</tabstop>
  <tabstop>lineEditCommandName</tabstop>
  <tabstop>checkBoxCmdToDevice1</tabstop>
  <tabstop>lineEditCLA</tabstop>
  <tabstop>lineEditINS</tabstop>
  <tabstop>lineEditP1</tabstop>
  <tabstop>lineEditP2</tabstop>
  <tabstop>lineEditLEN</tabstop>
  <tabstop>lineEditAnsLen</tabstop>
  <tabstop>lineEditAnsLenMin</tabstop>
  <tabstop>lineEditAnsLenMax</tabstop>
  <tabstop>lineEditSW1</tabstop>
  <tabstop>lineEditSW2</tabstop>
  <tabstop>checkSave2Buf1</tabstop>
  <tabstop>checkSave2Buf2</tabstop>
  <tabstop>checkLoadFromBuf1</tabstop>
  <tabstop>checkLoadFromBuf2</tabstop>
  <tabstop>checkBoxResetUSB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSetTestItem</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSetTestItem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
